rules/phasing.smk

```diff
@@ -4,16 +4,14 @@ rule phase:
             #idx="vcf/merged_mapped_sorted.vcf.gz.csi"
             vcfRef=REF_VCF
         output: temp("phase/chr{chrom}.phased.vcf.gz")
-        threads: 1
-        singularity:
-            "docker://biocontainers/bio-eagle:v2.4.1-1-deb_cv1"
+        threads: workflow.cores
```

Comment:
phasing works on each {chrom} separately, so there is no need for threads: workflow.cores. Snakemake will automatically launch parallel phasing jobs for {workflow.cores} chromosomes